### PR TITLE
Tt2 184 update logging to use aws lambda powertools

### PR DESCRIPTION
**Pull request by zaydgds**

### CODE REVIEW

File: src/lambdas/downloadWarning/handler.ts [hunk 1]
```diff
@@ -22,11 +28,13 @@ export const handler = async (
       return notFoundResponse()
     }
 
+    logger.info('Data persistence test log')
```
Comment (gdsandrewmccafferty) [inline src/lambdas/downloadWarning/handler.ts:31]: I presume you don't want to leave this in?

File: src/lambdas/generateDownload/writeOutSecureDownloadRecord.ts [hunk 2]
```diff
@@ -11,6 +12,7 @@ export const writeOutSecureDownloadRecord = async (parameters: {
   downloadHash: string
   zendeskId: string
 }) => {
+  logger.appendKeys({ zendeskId: parameters.zendeskId })
```
Comment (gdsandrewmccafferty) [inline src/lambdas/generateDownload/writeOutSecureDownloadRecord.ts:15]: We already have the Zendesk ID in the lambda handler, so I think you should do this there: the idea is to add data to the logger as soon as possible, so any other log lines that happen get that context.

File: template.yaml [hunk 3]
```diff
@@ -16,6 +16,7 @@ Parameters:
       - staging
       - integration
       - production
+    Default: dev
```
Comment (gdsandrewmccafferty) [inline template.yaml:19]: Any reason why this has been added? Don't think we had a need for a default before, because Github actions sort that out for us.

Comment (zaydgds) [reply template.yaml:19]: probably a result of the merge of ed's branch `origin/feature/powertools-logging`. will remove it

File: src/lambdas/generateDownload/writeOutSecureDownloadRecord.ts (hunk 2 above)
Comment (zaydgds) [reply src/lambdas/generateDownload/writeOutSecureDownloadRecord.ts:15]: moved it up to the handler

File: src/lambdas/downloadWarning/handler.ts (hunk 1 above)
Comment (zaydgds) [reply src/lambdas/downloadWarning/handler.ts:31]: removed

**Pull request closed without merging**